weaviate/collections/classes/grpc.py

```diff
@@ -153,13 +154,39 @@ class GroupBy(_WeaviateInput):
     number_of_groups: int
 
 
-class Sort(_WeaviateInput):
-    """Define how the query's sort operation should be performed."""
-
+class _Sort(_WeaviateInput):
     prop: str
     ascending: bool = Field(default=True)
 
 
+class Sort(_Sort):
+    """Define how the query's sort operation should be performed using the available static methods."""
+
+    def __init__(self, prop: str, ascending: bool = True) -> None:
+        _Warnings.sort_init_deprecated()
+        super().__init__(prop=prop, ascending=ascending)
+
+    @staticmethod
+    def by_property(prop: str, order: Literal["asc", "desc"] = "asc") -> _Sort:
```

Comment:
how about a bool?

Reply:
Yeh, I went back to it and thought that `ascending=False` being equivalent to `descending=True` to be odd. Andre also asked me how to use the previous `Sort` API and his instinct was to use `"asc"` instead of a bool

Reply:
maybe ask a few more people? My vote would be for a bool but if more people like "asc"/"desc" then that is how it is